stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +34,166 @@ class Worksheet(_WorkbookChild):
+        max_row: int | None = ...,
+        min_col: int | None = ...,
+        max_col: int | None = ...,
+        values_only: Literal[False] = False,
+    ) -> Generator[tuple[Cell, ...], None, None]: ...
+    @overload
+    def iter_rows(
+        self, min_row: int | None, max_row: int | None, min_col: int | None, max_col: int | None, values_only: bool
+    ) -> Generator[tuple[Cell | str | float | datetime | None, ...], None, None]: ...
+    @overload
+    def iter_rows(
+        self,
+        min_row: int | None = None,
+        max_row: int | None = None,
+        min_col: int | None = None,
+        max_col: int | None = None,
+        *,
+        values_only: bool,
+    ) -> Generator[tuple[Cell | str | float | datetime | None, ...], None, None]: ...
+    @property
+    def rows(self) -> Generator[Cell, None, None]: ...
+    @property
+    def values(self) -> Generator[str | float | datetime | None, None, None]: ...
+    @overload
+    def iter_cols(
+        self, min_col: int | None, max_col: int | None, min_row: int | None, max_row: int | None, values_only: Literal[True]
+    ) -> Generator[tuple[str | float | datetime | None, ...], None, None]: ...
+    @overload
+    def iter_cols(
+        self,
+        min_col: int | None = None,
+        max_col: int | None = None,
+        min_row: int | None = None,
+        max_row: int | None = None,
+        *,
+        values_only: Literal[True],
+    ) -> Generator[tuple[str | float | datetime | None, ...], None, None]: ...
+    @overload
     def iter_cols(
         self,
-        min_col: Incomplete | None = ...,
-        max_col: Incomplete | None = ...,
-        min_row: Incomplete | None = ...,
-        max_row: Incomplete | None = ...,
-        values_only: bool = ...,
-    ): ...
-    @property
-    def columns(self): ...
-    def set_printer_settings(self, paper_size, orientation) -> None: ...
-    def add_data_validation(self, data_validation) -> None: ...
+        min_col: int | None = ...,
+        max_col: int | None = ...,
+        min_row: int | None = ...,
+        max_row: int | None = ...,
+        values_only: Literal[False] = False,
+    ) -> Generator[tuple[Cell, ...], None, None]: ...
+    @overload
+    def iter_cols(
+        self, min_col: int | None, max_col: int | None, min_row: int | None, max_row: int | None, values_only: bool
+    ) -> Generator[tuple[Cell | str | float | datetime | None, ...], None, None]: ...
+    @overload
+    def iter_cols(
+        self,
+        min_col: int | None = None,
+        max_col: int | None = None,
+        min_row: int | None = None,
+        max_row: int | None = None,
+        *,
+        values_only: bool,
+    ) -> Generator[tuple[Cell | str | float | datetime | None, ...], None, None]: ...
+    @property
+    def columns(self) -> Generator[Cell, None, None]: ...
+    def set_printer_settings(
+        self, paper_size: int | None, orientation: None | Literal["default", "portrait", "landscape"]
```

Comment:
Another case for https://github.com/PyCQA/flake8-pyi/issues/175 and https://github.com/PyCQA/flake8-pyi/issues/283 -like checks in Flake8-PYI.